src/System.Text.Json/tests/Imported/CamelCaseTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization;
+using Xunit;
+
+namespace System.Text.Json.Tests.Imported
```

Comment:
What does "Imported" here mean?

If it means you copied tests from Json.NET, then we need to include the appropriate license information in the file and 3rd-party txt in the repo.

Otherwise, it'd be better to not include "Imported", and instead incorporate the tests into the test suite in an appropriate place, with appropriate naming, as if they were written from scratch (since that's what they are).

Reply:
> We do have Json.NET in the 3PN

Ok.  In files where we've incorporated code from the Json.NET repo, its license also needs to be incorporated into that file's license header, as we've done with code from Mono, for example.

Reply:
Yes I have cheked 3PN too

Tests are modified to be used for out new Json framework, also unit testing framework is different too. Not sure what level of licensing is required in that case...

I will change folder name, not all tests was importable due to not supported yet but as future features, we might want to import them after those features implemented. So the point of separating it from existing tests was to ease future imports (be able to find/distinguish not imported ones easily)

Reply:
> In files where we've incorporated code from the Json.NET repo, its license also needs to be incorporated into that file's license header, as we've done with code from Mono, for example.

I have a large .cs file I copied one string const from (one line of code) -- does that need a license header? If so, what's the purpose of adding the global 3PN? @richlander